client/blocks/get-apps/mobile-download-card.jsx

```diff
@@ -162,16 +169,84 @@ class MobileDownloadCard extends Component {
 		return this.props.hasUserSettings && this.props.hasLoadedAccountRecoveryPhone;
 	}
 
-	render() {
-		const { translate } = this.props;
+	getAppStoreBadges() {
+		const { isiPad, isiPod, isiPhone } = userAgent;
+		const isIos = isiPad || isiPod || isiPhone;
+
+		return (
+			<div className="get-apps__badges jetpack">
+				<p className="get-apps__card-text jetpack">
+					{ translate(
+						'Everything you need to publish, manage, and grow your site anywhere, any time.'
+					) }
+				</p>
+				<AppsBadge
```

Comment:
It would appear I overlooked updating the `title` for the `AppBadge` component in previous PRs. 😬 Sorry about that. Would you be willing to add a conditional text referencing "Jetpack?" It currently always says the "WordPress <iOS|Android> app." 

https://github.com/Automattic/wp-calypso/blob/2053b74187dbef03cb69a9dd1fd7cbe3b34e8629/client/blocks/get-apps/apps-badge.jsx#L28

https://github.com/Automattic/wp-calypso/blob/2053b74187dbef03cb69a9dd1fd7cbe3b34e8629/client/blocks/get-apps/apps-badge.jsx#L49